test/integration/targets/inventory_kubevirt_conformance/requirements.txt

```diff
@@ -0,0 +1,4 @@
+openshift
+
+# setuptools 45.0.0 no longer works with Python 2
+setuptools<45.0.0 ; python_version < '3'
```

Comment:
```suggestion
setuptools<45; python_version < "3.0"
setuptools; python_version >= "3.0"
```

Reply:
I've updated the first line.

Why should `setuptools` also be specified for Python >= 3.0? The test needs `openshift`; I only added `setuptools` because somewhere in the dependencies of `openshift`, a newer `setuptools` is mentioned than is currently installed.